packages/terra-search-field/src/terra-dev-site/doc/example/SearchFieldBasic.jsx

```diff
@@ -1,9 +1,38 @@
 import React from 'react';
 // eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved, import/extensions
-import SearchFieldExampleTemplate from 'terra-search-field/lib/terra-dev-site/doc/example/SearchFieldExampleTemplate';
+import SearchField from 'terra-search-field/lib/SearchField';
 
-const SearchFieldExample = () => (
-  <SearchFieldExampleTemplate />
-);
+class SearchFieldBasic extends React.Component {
+  constructor(props) {
+    super(props);
+    this.invalidSearchMessage = 'The defualt minimum search length is 2.';
+    this.state = { searchText: '', message: this.invalidSearchMessage };
+    this.handleSearch = this.handleSearch.bind(this);
+    this.handleInvalidSearch = this.handleInvalidSearch.bind(this);
+  }
 
-export default SearchFieldExample;
+  handleSearch(searchText) {
+    this.setState({ searchText, message: 'Search Text: ' });
```

Comment:
I think message as a state variable can be removed entirely. Here it is just being set to the same string on every state update. We can add a little logic into the render to display the different messages based on the searchText length.

length > 2 message otherwise display the invalid message